include/klee/Core/FunctionAnnotation.h

```diff
@@ -0,0 +1,46 @@
+#ifndef KLEE_FUNCTIONANNOTATION_H
+#define KLEE_FUNCTIONANNOTATION_H
+
+#include <vector>
+#include <map>
+#include <string>
+
+#include <nonstd/optional.hpp>
+#include <nlohmann/json.hpp>
+
+using nonstd::optional;
+using nonstd::nullopt;
+using json = nlohmann::json;
+
+namespace klee {
+
+struct FunctionAnnotation {
```

Comment:
Add comment with link to discussion of annotations

Reply:
Done.